readme.md

```diff
@@ -1,30 +1,63 @@
-# WIP: @jspm/node-importmap-loader
+# @jspm/node-importmap-loader
 
-##### Don't bother installing dependencies you don't need to! 🏇🏻💨
+![Typed with TypeScript](https://flat.badgen.net/badge/icon/Typed?icon=typescript&label&labelColor=blue&color=555555)
+[![npm version](https://badge.fury.io/js/node-importmap-http-loader.svg)](https://badge.fury.io/js/node-importmap-http-loader)
+![ci](https://github.com/yowainwright/node-importmap-http-loader/actions/workflows/ci.yml/badge.svg)
+[![Github](https://badgen.net/badge/icon/github?icon=github&label&color=black)](https://github.com/yowainwright/node-importmap-http-loader)
 
-With **JSPM _node-importmap-loader_**, installing dependencies is a thing of the past! You can reference and execute them directly! Maximize productivity ⚡️, decrease security risk 🚔, and probably do other awesome things too! Use **JSPM _node-importmap-loader_** today!
+#### Don't bother installing dependencies you don't need to! 🏇🏻💨
 
-##### I'm skeptical, is this for real?
+With `@jspm/node-importmap-loader`, installing node modules is a thing of the past!
```

Comment:
Something like this, we still reply on `node_modules` to use the loader from them. So it's raises unnecessary questions from useers.